### PR TITLE
home page, feed page, login 

### PR DESCRIPTION
**Pull request by mutahharaj**
worked on the feed  home page, feed page, login

### CODE REVIEW

File: src/assets/Novelology_Logo.png
Comment (racecn) [inline src/assets/Novelology_Logo.png:1]: Can we create some .SVG formatted files so we have transparency and can change the colors as needed?

File: src/App.js
Comment (racecn) [inline src/App.js:1]: tailwind height can be implemented as h-[200px] or h-[2em] if you'd like to make typing easier on yourself

File: src/components/BookSearch.js
Comment (racecn) [inline src/components/BookSearch.js:1]: Custom colors in tailwind can be implemented into 'tailwind.config.js' as follows:

module.exports = {
  theme: {
    extend: {
      colors: {
        'primary': '#243c5a',
      },
    }
  }
}

File: src/components/SignIn.js
Comment (racecn) [inline src/components/SignIn.js:1]: We are mixing external style sheets like SignIn.css, inline CSS like style={{}} AND tailwind here which can make it confusing to fix down the line

File: src/pages/Home.js
Comment (racecn) [inline src/pages/Home.js:1]: If you want to move stuff you can just delete the old code out instead of commenting it out

File: src/styles/App.css
Comment (racecn) [inline src/styles/App.css:1]: This background color seems dark on the lighter text and borders

File: src/assets/Novelology_Logo.png
Comment (rishithab48) [reply src/assets/Novelology_Logo.png:1]: That's a good idea. We can style it later if we need to.

**Review by hello202320 — APPROVED** [review]

**Pull request merged**